src/index.html

```diff
@@ -11,6 +11,37 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form class="form1"
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    data-qa="big"
+>
+      <input
+        class="input1"
+        type="text"
+        placeholder="`Try &quot;Los Angeles&quot;`"
+        data-qa="keypress"
+     >
+      <svg
+        class="img1">
+      </svg>
+    </form>
+    <form
+    class="form2"
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post"
+    data-qa="small"
+    >
+      <input
+        class="input2"
+        type="text"
+        placeholder="`Try &quot;Los Angeles&quot;`"
+       >
+      <svg
+       class="img2">
+      </svg>
+
+    </form>
   </body>
+
 </html>
```

Comment:
```suggestion
      </svg>
    </form>
  </body>
</html>
```
use empty line for same-level elements